ts/Grid/Pro/ColumnTypes/Validator.ts

```diff
@@ -123,18 +124,66 @@ class Validator {
             },
             notification:
                 'Value must be unique within this column (case-sensitive).'
+        },
+        arrayNumber: {
+            validate: function ({ rawValue }): boolean {
+                if (typeof rawValue !== 'string') {
+                    return false;
+                }
+
+                return rawValue
+                    .split(',')
+                    .every(
+                        (item): boolean => !Number.isNaN(Number(item.trim()))
+                    );
+            },
+            notification:
+                'Value should be a list of numbers separated by commas.'
+        },
+        json: {
+            validate: function ({ rawValue }): boolean {
+                try {
+                    JSON.parse(rawValue);
+                    return true;
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                } catch (e) {
```

Comment:
`e` is not used anywhere, so you can remove it from here.

Reply:
Without `e` syntax isn't correct, it has to stay line this